SillyBusinessInc/SillyBusinessGame
# Add support for manual enemies

## Purpose of this PR:
This script modiefies the Enemy prefab and adds behavior for manual enemy placement

## How to test:
In the scene Assets/Dev/EnemySpawner/EnemySpawner.scene you can click play and kill the enemy by selecting the enemy and in the context menu of the followingEnemy script click "DIE". After all enemies are killed the console should confirm that the wave is over as the WAVE_DONE event is called.

### links: 
[Ticket](https://github.com/orgs/SillyBusinessInc/projects/1/views/1?pane=issue&itemId=87056613&issue=SillyBusinessInc%7CSillyBusinessGame%7C54)
